File: python/raindrops/raindrops.py
def convert(number):
    sound = ""
    if number % 3 == 0:
        sound += "Pling"
    if number % 5 == 0:
        sound += "Plang"
    if number % 7 == 0:
        sound += "Plong"
    if sound:
        return sound
    return str(number)


if __name__ == "__main__":
    print(f"{convert(1)=}")
    print(f"{convert(2)=}")
    print(f"{convert(3)=}")
    print(f"{convert(4)=}")
    print(f"{convert(5)=}")
    print(f"{convert(6)=}")
    print(f"{convert(15)=}")
    print(f"{convert(105)=}")

File: go/raindrops/raindrops.go
// Package raindrops implements a solution for the exercism raindrops chalenge
package raindrops

import (
	"strconv"
)

// Convert returns the "raindrop" string corresponding to the factors of the integer arg
func Convert(num int) string {
	result := ""
	if num%3 == 0 {
		result += "Pling"
	}
	if num%5 == 0 {
		result += "Plang"
	}
	if num%7 == 0 {
		result += "Plong"
	}
	if len(result) == 0 {
		result += strconv.Itoa(num)
	}
	return result
}
